src/OptionT.ts

```diff
@@ -63,18 +81,18 @@ export interface OptT<T> {
    * ```
    * const maybeOne = OptionT.some(1);
    * // this won't throw, because it's a Some value.
-   * const one = maybeOne.expect("couldn't unwrap a Some");
+   * const one = maybeOne.expect('couldn't unwrap a Some');
    *
    * // but:
    * const maybeTwo = OptionT.none();
    * // this will throw, because it's a None value.
-   * const two = maybeTwo.expect("couldn't unwrap a Some");
+   * const two = maybeTwo.expect('couldn't unwrap a Some');
```

Comment:
nitpick here: I think these need to be `"`s instead of `'`s because of the `'` in "couldn't".